client/app/components/packages/landuse-form/project-area.hbs

```diff
@@ -0,0 +1,159 @@
+{{#let @form as |form|}}
+  <form.Section @title="Proposed Project Area">
+    <p>
+      The <b>Project Area</b> is the entirety of all land parcels
+      that are affected by all actions being sought.
+    </p>
+
+    <Ui::Question as
+    |dcpWholecityQ|>
+      <dcpWholecityQ.Label>
+        Do <em>all</em> Actions being sought apply to the whole City?
+      </dcpWholecityQ.Label>
+
+      <form.Field
+        @attribute="dcpWholecity"
+        @type="radio-group"
+        as |DcpWholeCityRadioGroup|
+      >
+        <DcpWholeCityRadioGroup
+          @options={{optionset 'landuseForm' 'dcpWholecity' 'list'}}
+          @onChange={{fn this.resetDependentFields this.dcpWholecity}}
+        as |dcpWholecity|>
+          {{#if (eq
+            dcpWholecity
+            (optionset 'landuseForm' 'dcpWholecity' 'code' 'NO')
+          )}}
+
+            <Ui::Question as
+            |dcpEntiretyboroughsQ|>
+              <dcpEntiretyboroughsQ.Label>
+                Do <em>all</em> Actions being sought apply to the entirety of one or
+                more Boroughs?
+              </dcpEntiretyboroughsQ.Label>
+
+              <form.Field
+                @attribute="dcpEntiretyboroughs"
+                @type="radio-group"
+                as |DcpEntiretyboroughsRadioGroup|
```

Comment:
The naming here is awkward. What about `IsAllBoroughsRadioGroup`?

Reply:
Hmm, I wanted to stick with the format of `<field_name>RadioGroup`, instead of creating a new alias, so that the component names are predictable for developers.